src/services/project.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { Project } from '../models/project.model';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ProjectService {
+    private project: Project[] = [];
+
+    addProject(name: string) {
```

Comment:
Add access modifier and return type to all your functions
```suggestion
    public addProject(name: string): void {
```